guides/PlayingOnDev.md

```diff
@@ -46,21 +46,25 @@ Simply download the client zip and import it into any modded Minecraft launcher
 
 Note that in May 2022, CurseForge switched to a new API that prevents third-party launchers (all but FTB and Curse) from downloading mods through the API unless that mod's author opted-in to allowing said downloads. Some launchers have implemented workarounds, while others remain incapable of installing packs unless every single mod permits third-party downloads:
 
-![launchers](https://cdn.discordapp.com/attachments/927073124086849577/1037904047614726205/Nomifactory_Infographics_-_Launcher_Import_Zip5.jpg)
+![launchers](https://github.com/rogerwim/Guides/assets/33426007/49217e32-5845-41f0-947a-773032cee60e)
```

Comment:
as discussed let's upload the actual file to an assets directory or something, then use a relative link. I worry this link generated by drag-and-drop into the editor will break in the future.

Reply:
fixex